enable_dnf5.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/bash
+
+dnf -y copr enable rpmsoftwaremanagement/dnf5-testing
+dnf -y install dnf5
+dnf -y update dnf5 --repo=copr:copr.fedorainfracloud.org:rpmsoftwaremanagement:dnf5-testing
```

Comment:
Why do we need the update? Doesn't the `dnf -y install dnf5` install the latest RPM?
If you think it's needed, please add a comment above this (`dnf -y update dnf5 ...`) line, thanks.

Reply:
Without the update it pulls the package from fedora repos, not from Copr. I'll add a comment